stand-alone templates/laureate-template.html

```diff
@@ -0,0 +1,554 @@
+<!--This is where we define the CSS styles to be later used in the page content-->
+<style>
+  
+  .custom-card-group {
+    padding: 24px;
+    border: 1px solid var(--mp-color-primary-100);
+  }
+  
+  .custom-card-content {
+    flex: 1 1 auto;
+    overflow: hidden;
+    height: 120px;
+    padding: 16px 8px 8px 0px;
+  }
+  
+  .DataTable--semantic-data-table {
+    position: relative;
+    height: auto;
+  }
+  
+  .custom-page-header {
+    padding: 16px 64px 16px 78px;
+    background-color: var(--mp-color-dark-50);
+    display: flex;
+    align-items: start;
+  }
+  
+  .custom-page-content {
+    display: flex;
+    align-items: start;
+  }
+  
+  .main-content {
+    padding: 8px 96px 8px 24px;
+    width: 100%;
+    border-top: 1px solid var(--mp-color-primary-100);
+  }
+  
+  .navigationSidebar {
+    width: 310px;
+    margin-left: 75px;
+    border: 1px solid var(--mp-color-primary-100);
+    z-index: 100;
+    position: sticky;
+    top: 70px;
+    display: flex;
+    flex-direction: column;
+    padding: 16px;
+    line-height: 2.5em;
+  }
+  
+  .page-nav-header {
+    margin-top: 8px;
+    padding-bottom: 8px;
+    border-bottom: 1px solid var(--mp-color-primary-100);
+  }
+  
+  .page-nav-button {
+    width: 265px;
+    padding: 0;
+    font-size: 12px;
+  }
+  
+  .additional-information {
+    display: grid; 
+    padding-top: 8px; 
+    grid-template-columns: repeat(2, 1fr); 
+    gap: 10px;
+    width: 50%;
+  }
+  
+  ul {
+    list-style: none;
+  }
+  
+  ul li::before {

```

Comment:
Please remove the pseudo element on `ul` as we are not using it anywhere in the template.